fav_and_rating.py

```
@@ -0,0 +1,139 @@
+
+import os
+from pathlib import Path
+
+from dotenv import load_dotenv
+from PySide6.QtCore import QSize
+from PySide6.QtGui import QIcon, QPixmap
+from PySide6.QtWidgets import QHBoxLayout, QLabel, QPushButton, QSizePolicy, QWidget
+
+load_dotenv()
+resources_path = os.getenv("FRONTEND_RESOURCES")
+IMAGES = Path((resources_path or ""))
```

Ruff (0.15.20)

[warning] Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)